File: stress/t/flt_saxpy.f
      PROGRAM FSAXPY
      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
      INTEGER*8 I,N
      DIMENSION X(0:63),Y(0:63)
      N=40
      A=2.5D0
      DO 10 I=0,N-1
        X(I)=DBLE(I)*0.25D0
        Y(I)=DBLE(I)+1.0D0
   10 CONTINUE
      DO 20 I=0,N-1
        Y(I)=A*X(I)+Y(I)
   20 CONTINUE
      SUM=0.0D0
      DO 30 I=0,N-1
        T=Y(I)
        SUM=SUM+T
   30 CONTINUE
      WRITE(*,'(A,F0.6)') 'sum= ',SUM
      END

File: stress/t/flt_saxpy.c
#include <stdio.h>
#include <math.h>
int main(void){
  long i,n=40; double a=2.5,sum,t; double x[64],y[64];
  for(i=0;i<n;i++){ x[i]=(double)i*0.25; y[i]=(double)i+1.0; }
  for(i=0;i<n;i++){ y[i]=a*x[i]+y[i]; }
  sum=0.0;
  for(i=0;i<n;i++){ t=y[i]; sum=sum+t; }
  printf("sum= %f\n",sum);
  return 0;
}
